node-manager/src/node.rs

```diff
@@ -108,7 +108,7 @@ impl PubkeyConnectionInfo {
                 .context("connect_peer requires peer connection info")?;
         };
 
-        if connection.starts_with("mutiny:") {
+        if connection.to_lowercase().starts_with("mutiny:") {
             let stripped_connection = connection.strip_prefix("mutiny:").expect("should strip");
```

Comment:
I think this strip_prefix will fail if it isn't all lowercase now

Reply:
good catch, doing it at the beginning now so it should apply to both